pages/omnibar.js

```diff
@@ -995,9 +995,13 @@ var SearchEngine = (function() {
         return this.activeTab;
     };
     self.onInput = function() {
-        if (!self.suggestionURL || typeof(self.listSuggestion) !== "function") {
-            return;
-        }
+        const showSuggestions = (
+            !self.suggestionURL
```

Comment:
Looks like the expression here is wrong, please make sure test done.

Reply:
Can you provide more detail?
I have tested this on my machine and it is working correctly.

Reply:
Should it be 

    const showSuggestions = (
                self.suggestionURL
                && typeof(self.listSuggestion) == "function")
                && runtime.conf.omnibarSuggestion
    

With your change, when `omnibarSuggestion` is true(and I prefer it default to be true, as most users are used to this behavior.), no suggestion will be listed, please notice that you're using `!showSuggestions` below.

Reply:
You're right. Thanks for your input! Sending the pull request now.